backend/tests/services/test_task_member_service_group_chat.py

```
@@ -177,12 +177,15 @@ def test_get_members_excludes_share_records(self, task_member_service, mock_db):
-        with patch.object(
-            task_member_service, "get_task", return_value=mock_task
-        ), patch.object(
-            task_member_service,
-            "get_user",
-            side_effect=lambda db, uid: mock_owner if uid == 1 else mock_member_user,
+        with (
+            patch.object(task_member_service, "get_task", return_value=mock_task),
+            patch.object(
+                task_member_service,
+                "get_user",
+                side_effect=lambda db, uid: (
```

Ruff (0.15.4)

[warning] Unused lambda argument: `db`

(ARG005)